README.md

```diff
@@ -1,20 +1,65 @@
 # SKALE Network
+ <img src="https://github.com/user-attachments/assets/e25b683d-874b-478a-9964-b589076d749c" width="300" height="300" style="border:1px solid #ccc"/>
 
-## Table of Contents
+The SKALE Network is an open-source, Ethereum-native multichain network built to scale Ethereum dApps with a focus on high-throughput, fast finality, and **zero gas fee** transactions via the native gas token sFUEL.
 
-- [Open Source Plan and Community](OPEN_SOURCE_PLAN.md)
+## What does SKALE offer?
+
+- **Zero gas fees** - by design, developers and end-users never pay for transactions.
+- **Instant finality** - the fastest chain to finality. Time to finality = blocktime.
+- **High performance**
+  - **Throughput** - Baseline of 400 transactions per second (tps) and up to 700 tps at peak.
+  - **High block gas limit** - 280 million block gas limit allowing optimal throughput and complex transactions/smart contracts to be executed.
+- **Purpose-built chains** - app-chains and hub-chains for customization and and horizontal scaling.
+- **Native bridge** - Fast flexible, offering 18 second finality between chains. Direct support for popular token standards such as ERC-20, ERC-721, and ERC-1155 with additional flexibility for passing any arbitrary data through the low-level messaging protocol.
+- **Native oracle** - decentralized and transaction free oracle enabling developers to read off-chain data.
```

Comment:
Proposed Text: a native and fully decentralized oracle is available on sChains which is fully capable of reading smart contracts of other EVM chains and offchain public data